includes/admin/tools.php

```diff
@@ -1467,6 +1468,29 @@ function edd_tools_sysinfo_get() {
 	return $return;
 }
 
+/**
+ * Get TLS Info
+ *
+ * @since       2.9
+ * @return      string
+ */
+function edd_tools_tlsinfo( $fail_response ) {
+
+	$tls_info = false;
+
+	if ( ini_get( 'allow_url_fopen' ) ) {
+		$tls_info = file_get_contents( 'https://www.howsmyssl.com/a/check' );
```

Comment:
@SimonChaw Just noting that file_get_contents is also discouraged by WPCS. They still recommend wp_remote_get here:

https://github.com/WordPress/WordPress-Coding-Standards/blob/fab1ed525e057c19d723bf7c93feaf0a48512abe/WordPress/Sniffs/WP/AlternativeFunctionsSniff.php#L114